code/modules/keybindings/bindings_atom.dm

```diff
@@ -1,17 +1,21 @@
 // You might be wondering why this isn't client level. If focus is null, we don't want you to move.
 // Only way to do that is to tie the behavior into the focus's keyLoop().
 /atom/movable/keyLoop(client/user)
-	if(!user.keys_held["Ctrl"])
-		var/movement_dir = NONE
-		for(var/_key in user.keys_held)
-			movement_dir = movement_dir | SSinput.movement_keys[_key]
-		if(user.next_move_dir_add)
-			movement_dir |= user.next_move_dir_add
-		if(user.next_move_dir_sub)
-			movement_dir &= ~user.next_move_dir_sub
-		// Sanity checks in case you hold left and right and up to make sure you only go up
-		if((movement_dir & NORTH) && (movement_dir & SOUTH))
-			movement_dir &= ~(NORTH|SOUTH)
-		if((movement_dir & EAST) && (movement_dir & WEST))
-			movement_dir &= ~(EAST|WEST)
-		user.Move(get_step(src, movement_dir), movement_dir)
+	if(user.keys_held["Ctrl"])
```

Comment:
This is something I'd like to change, by the byes.
The ability to turn around holding CTRL is a bit less interesting than the ability to use it for secondary actions, such as grabbing someone or firing from an attachment while moving.

Reply:
I agree, i just didn't change it because a lot of people reee'd when i broke it.

Reply:
Just revert the indentation changes.